packages/storybook/.storybook/main.js

```diff
@@ -24,7 +24,7 @@ module.exports = {
     '@storybook/addon-a11y/register',
   ],
   typescript: {
-    check: __DEV__, // Netlify is breaking the deploy with this settings on. So deactivate on release
+    check: __DEV__, // Already covered by yarn typescript
```

Comment:
You want it at compile time not in the CI. We develop in typescript not in JS and then apply some nice tools to make it nice. We want to see errors straightaway and not put ts-ignore when it does not work

Reply:
Bundling and type static analysis are two different concerns. The new `yarn typescript` solve the type static analysis part of the problem. It's a lot faster to run than building. Not running the checks when running for production avoid doing the static analysis twice in the CI making it faster.